src/Sylius/Bundle/ApiBundle/OpenApi/Documentation/PromotionDocumentationModifier.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Sylius\Bundle\ApiBundle\OpenApi\Documentation;
+
+use ApiPlatform\OpenApi\Model\Paths;
+use ApiPlatform\OpenApi\OpenApi;
+
+/** @experimental */
+final class PromotionDocumentationModifier implements DocumentationModifierInterface
+{
+    /**
+     * @param string[] $actionTypes
+     * @param string[] $ruleTypes
+     */
+    public function __construct(
+        private string $apiRoute,
+        private array $actionTypes,
+        private array $ruleTypes,
+    ) {
+    }
+
+    public function modify(OpenApi $docs): OpenApi
+    {
+        $paths = $docs->getPaths();
+
+        $this->addDescription($paths, sprintf('%s/admin/promotions', $this->apiRoute), 'Post');
+        $this->addDescription($paths, sprintf('%s/admin/promotions/{code}', $this->apiRoute), 'Put');
```

Comment:
```suggestion
        $this->addDescription($paths, sprintf('%s/admin/promotions', $this->apiRoute), 'Post');
        $this->addDescription($paths, sprintf('%s/admin/promotions/{code}', $this->apiRoute), 'Put');
```

For me, it looks like a good place to use constants.